sdk/spring/README.md

```diff
@@ -148,6 +148,31 @@ If you’re a Maven user, add our BOM to your pom.xml `<dependencyManagement>` s
 ```
 [//]: # ({x-version-update-end})
 
+With Gradle, you can import the `spring-cloud-azure-dependencies` BOM in [two ways](https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/htmlsingle/).
+
+You can use the Gradle’s native BOM support by adding dependencies:
+
+```kotlin
+dependencies {
+    implementation(platform(SpringBootPlugin.BOM_COORDINATES))
```

Comment:
![image](https://github.com/Azure/azure-sdk-for-java/assets/92105726/c6c0bd0a-016d-4089-bbee-ba3e89ab8c64)

I meet this error when adding the dependency, do I overlook something?

Reply:
It is defined in the Spring Boot  Gradle plugin. I have updated the example.